plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -134,6 +142,8 @@ contract RootChain {
         bytes32 root = childChain[blknum].root;
 
         require(msg.sender == txList[6 + 2 * oindex].toAddress());
+        require(msg.value == minExitBond);
```

Comment:
Should we require `msg.value == minExitBond` or simply send back any excess?

Reply:
minExitBond is a public variable so it should be simple for users to simply pass in the exact amount as the value of the transaction.

If we send back excess here, I think we should do that in deposit as well to maintain consistency.

Reply:
Yeah, I think this is fine as-is. There's a possibility that it might become useful to send back excess in the case that `minExitBond` changes frequently, but I'd rather stick to simplicity for now.